utils/validators/validators.go

```diff
@@ -71,3 +71,9 @@ func IsETHAddress(str string) bool {
 	}
 	return altcurrency.ToChecksumETHAddress(str) == str
 }
+
+// IsPlatform determines whether or not a given string is a recognized platform
+func IsPlatform(platform string) bool {
+	platforms := []string{"ios", "android", "osx", "windows", "linux"}
```

Comment:
do we need per OS or can we just say desktop?

Reply:
probably not needed for restricting grants but having separated desktop platforms helps for DAU might be useful